lib/apipie/extractor/writer.rb

```diff
@@ -3,11 +3,76 @@
 module Apipie
   module Extractor
     class Writer
+      class << self
```

Comment:
@Stex Most of those are just functions that were defined with `def self.` before.
I moved them so that encapsulation actually works.